lib/suppressions.cpp

```diff
@@ -373,6 +373,7 @@ void Suppressions::dump(std::ostream & out) const
 
 std::list<Suppressions::Suppression> Suppressions::getUnmatchedLocalSuppressions(const std::string &file, const bool unusedFunctionChecking) const
 {
+    std::string tmpFile = Path::simplifyPath(file);
```

Comment:
well.. it would be better with a `const` variable.. but I don't want to nit too much. I'll merge this as is. Please use `const` in the future.

Reply:
Actually, at the moment, as many guidelines and notes as possible is nice for better code in the future. As long as the comments are friendly. :-)